lab-taylor/routes/gallery-router.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const Router = require('express').Router;
+const jsonParser = require('body-parser').json();
```

Comment:
express actually has a built in body parser, so you could do something like this:

```
const express = require('express');
const { json, Router } = express;
```

Now you have them w/o an extra dependency. However, notice that you haven't called `json`, so you would either need to call it and store it in a variable, or use it as `json()` in your route level middleware.